askama_parser/src/expr.rs

```diff
@@ -1144,7 +1128,7 @@ fn doc_comment_no_bare_cr<'a>(
     let (is_doc_comment, comment) = inner.with_taken().parse_next(i)?;
     if is_doc_comment && comment.split('\r').skip(1).any(|s| !s.starts_with('\n')) {
         cut_error!(
-            "bare CR not allowed in doc comment, 
+            "bare CR not allowed in doc comment,
```

Comment:
Oops, looks like I forgot to add a backslash in this message, and the editor (rightfully so) trimmed trailing spaces. I'll fix this is another PR and add the missing UI test that should have caught this change.